gtest/src/manager.rs

```diff
@@ -419,6 +422,17 @@ impl ExtManager {
 
         core_processor::handle_journal(journal, self);
     }
+
+    pub(crate) fn get_mailbox(&self, program_id: &ProgramId) -> Option<&Mailbox>{
+        self.actor_to_mailbox.get(program_id)
+    }
+
+    pub(crate) fn remove_message(&mut self, program_id: &ProgramId, message_id: &MessageId) -> Result<Message, String>{
+        match  self.actor_to_mailbox.get_mut(program_id) {
+            None => {Err(String::from("No actor with such program id"))}
+            Some(mailbox) => { mailbox.remove_message(message_id)}
+        }
+    }
```

Comment:
This should always return mailbox: empty or not. And panic in case of program id